File: Q15 Maximum point you can obtain from a card.java
class Solution {
    public int maxScore(int[] cardPoints, int k) {
        int ans = 0;
        int n = cardPoints.length;
        for (int i = 0; i < k; i++) {
            ans += cardPoints[i];
        }
        int sum = ans;
        for (int i = 0; i < k; i++) {
            sum = sum - cardPoints[k - i - 1] + cardPoints[n - i - 1];
            ans = Math.max(ans, sum);
        }
        return ans;
    }
}

File: Q15 Maximum point you can obtain from a card.js
var maxScore = function(cardPoints, k) {
    let ans = 0;
    const n = cardPoints.length;
    for (let i = 0; i < k; i++) {
        ans += cardPoints[i];
    }
    let sum = ans;
    for (let i = 0; i < k; i++) {
        sum = sum - cardPoints[k - i - 1] + cardPoints[n - i - 1];
        ans = Math.max(ans, sum);
    }
    return ans;
};
